src/npc.h

```diff
@@ -532,7 +532,7 @@ struct npc_short_term_cache {
     int stuck = 0;
     // Position to return to guarding
     cata::optional<tripoint> guard_pos;
-    double my_weapon_value = 0;
+    double my_weapon_value = 0.0f;
```

Comment:
What? How is that an "optimization". Instead of initialising a `double` with an `int`, you initialise it with a `float`.

0 is 0 regardless of the type, the conversion from `int(0)` to `float(0)`, to `double(0)` is unambiguous. How is this better.

Reply:
It is not - I only wanted to use `f` literal for floats, not for doubles. It is fixed now.